setup:
    prefix: 'j2s7s300_driver'
    model_filename: "jaco_dynamics"
    object_centers: {'HUMAN_CENTER': [-0.6,-0.55,0.0], 'LAPTOP_CENTER': [-0.7929,-0.1,0.0]}
    # for the features and weights that are shared between all goals
    common_feat_list: ['efficiency']
    common_feat_weights: [1.0]
    #start: [180.0, 160.0, 175.0, 40.0, 275.0, 180.0, 280.0]
    #goals: [
    #    [160.0, 90.0, 180.0, 160.0, 270.0, 180.0, 280.0],
    #    [200.0, 90.0, 180.0, 160.0, 270.0, 180.0, 280.0]
    #]
    start: [180.0, 150.0, 180.0, 270.0, 180.0, 180.0, 190.0]
    # for generating goal-specific feature weights
    goals: [
        #[140.0, 90.0, 180.0, 160.0, 270.0, 180.0, 280.0],
        #[220.0, 90.0, 180.0, 160.0, 270.0, 180.0, 280.0],
        [150.0, 130.0, 160.0, 240.0, 180.0, 172.0, 280.0],
        [210.0, 130.0, 200.0, 240.0, 180.0, 172.0, 280.0]
    ]
    goal_dist_feat_weight: 0.0
    T: 20.0
    timestep: 0.5
    save_dir: "/data/teleoperation/"

planner:
    # These settings have been tuned for trajopt planner.
    type: "trajopt"
    max_iter: 50
    num_waypts: 5

controller:
    # These settings have been tuned for PID controller.
    type: "pid"
    p_gain: 0.5
    i_gain: 0.0
    d_gain: 0.1
    epsilon: 0.10
    max_cmd: 40.0

learner:
    betas: [0.1, 1, 10.]
    goal_beliefs: [0.5, 0.5, 0.5] #just for debugging, if there are saved goals then uniform gets used
    #inference_method: "dragan"
    inference_method: "none"
    #assistance_method: "blend"
    assistance_method: "none"
    beta_method: "estimate"
    #beta_priors: [0.4, 0.4, 0.2] # if "joint" these are probabilities
    beta_priors: [2, 2] #if "estimate" these are exponential rate parameters
